### PR TITLE
[zh] sync kubelet-garbage-collection.md

### PR DESCRIPTION
**Pull request by Arhell**
sync PR https://github.com/kubernetes/website/pull/25457

### CODE REVIEW

File: content/zh/docs/concepts/cluster-administration/kubelet-garbage-collection.md [hunk 1]
```diff
@@ -1,25 +1,24 @@
 ---
-title: 配置 kubelet 垃圾回收策略
+title: 容器图像的垃圾收集
```
Comment (tengqm) [inline content/zh/docs/concepts/cluster-administration/kubelet-garbage-collection.md:2]:
```suggestion
title: 容器镜像的垃圾收集
```

File: content/zh/docs/concepts/cluster-administration/kubelet-garbage-collection.md [hunk 2]
```diff
@@ -1,25 +1,24 @@
...
 <!--
-Garbage collection is a helpful function of kubelet that will clean up unused images and unused containers. 
-Kubelet will perform garbage collection for containers every minute and garbage collection for images every five minutes.
+Garbage collection is a helpful function of kubelet that will clean up unused [images](/docs/concepts/containers/#container-images) and unused [containers](/docs/concepts/containers/). Kubelet will perform garbage collection for containers every minute and garbage collection for images every five minutes.
 
 External garbage collection tools are not recommended as these tools can potentially break the behavior of kubelet by removing containers expected to exist.
 -->
 
-垃圾回收是 kubelet 的一个有用功能，它将清理未使用的镜像和容器。
+垃圾回收是 kubelet 的一个有用功能，它将清理未使用的镜[像](/zh/docs/concepts/containers/#container-images)和[容器](/zh/docs/concepts/containers/)。
```
Comment (tengqm) [inline content/zh/docs/concepts/cluster-administration/kubelet-garbage-collection.md:21]:
```suggestion
垃圾回收是 kubelet 的一个有用功能，它将清理未使用的[镜像](/zh/docs/concepts/containers/#container-images)和[容器](/zh/docs/concepts/containers/)。
```

File: content/zh/docs/concepts/cluster-administration/kubelet-garbage-collection.md (hunk 1 above)
Comment (Arhell) [reply content/zh/docs/concepts/cluster-administration/kubelet-garbage-collection.md]: Done
Thank you

File: content/zh/docs/concepts/cluster-administration/kubelet-garbage-collection.md (hunk 2 above)
Comment (Arhell) [reply content/zh/docs/concepts/cluster-administration/kubelet-garbage-collection.md]: Done
Thank you

**Pull request merged**